shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -63,21 +63,24 @@
                                                        accessibility_bridge_.reset();
                                                        owner_controller_.reset();
                                                      }] retain]);
+    
```

Comment:
We could check here if the view controller `isViewLoaded`, just immediately call `attachView` - I'm not sure if that path is ever possible currently, but it seems like it would be easy to create and then miss.